### PR TITLE
DIRECTOR: Add detection entries for Tivola Sampler discs

### PR DESCRIPTION
**Pull request by lotharsm**
This PR adds a detection entry for the German sampler disc of Tivola, released in Summer 2000.

It contains various samples and demos of other DIRECTOR games made by the same publisher that can be launched through the START32/START16.EXE projects.

Reported DIRECTOR version according to the file details is 6.5G.

### CODE REVIEW

**Comment by lotharsm** [comment]
I received 3 more sampler discs from the same publisher, I'll hopefully add them later today.

File: engines/director/detection_tables.h [hunk 1]
```diff
@@ -7785,6 +7786,9 @@ static const DirectorGameDescription gameDescriptions[] = {
 
 	WINGAME1("thesims", "", "maxis.exe", "d62438566e44826960fc16c5c23dbe43", 1915533, 650),
 
+	// Tivola demo/sampler disc, Summer 2000, released in Germany
+	WINGAME1_l("tivolasummer2000", "Sampler", "START32.EXE", "1b8d78ddca650041b8997cac7af3184b", 1675134, Common::DE_DEU, 650),
```
Comment (sev-) [inline engines/director/detection_tables.h:7790]: For Windows titles, please use tail md5 checksum. The beginning of the file is always the same.

**Comment by lotharsm** [comment]
Any objections of merging this even though not all discs are in yet? As usual, I got distracted when starting this and now I don't want to create conflicts considering that there's ongoing work on the detection table.

**Comment by tag2015** [comment]
Just two nitpicks:
- If this is a german-only title, you should add it to the "German Titles" group instead of the Promos
- If you add `d: ` to the second file checksum it lines up with the other one and it looks cleaner

**Comment by lotharsm** [comment]
@tag2015 The thing is that it is _both_ a promo and a German title... I don't really know what's more important here...

What exactly does the `d:` flag do?

**Comment by tag2015** [comment]
> @tag2015 The thing is that it is _both_ a promo and a German title... I don't really know what's more important here...
Titles that contain only non-English languages generally are included in their own group (German Titles / Dutch Titles etc) even if they are promos/demos. But use your judgment, it's just indicative
> What exactly does the `d:` flag do?
Simply indicates that the checksum is calculated on head

**Comment by sev-** [comment]
Thank you!

**Pull request merged**